### PR TITLE
Update `DOMString` to `string`

### PR DESCRIPTION
**Pull request by OnkarRuikar**
### Summary
`DOMString` is not exposed by browsers to JavaScript. It is converted to `string`.[[ref](https://developer.mozilla.org/en-US/docs/Web/API/DOMString)]

Same is the case with `USVString` and `CSSOMString`.

### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks @OnkarRuikar - So we should NEVER ever use DomString anywhere? I've just been blindly copying it :-(

**Comment by OnkarRuikar** [comment]
> So we should NEVER ever use DomString anywhere?

If developers can't use these interfaces, then there is no point in suggesting them:
> not available

We can mention these in "See also" sections of their own pages: https://developer.mozilla.org/en-US/docs/Web/API/DOMString#see_also

File: files/en-us/mozilla/firefox/releases/18/index.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ Firefox 18 was released on January 8, 2013. This article lists key changes that
 - {{domxref("BlobBuilder", "MozBlobBuilder")}} is removed. Developers need to use {{domxref("Blob")}} constructor for creating a `Blob` object. ({{bug("744907")}})
 - The {{domxref("document.visibilitychange_event", "visibilitychange")}} event and the [Page Visibility API](/en-US/docs/Web/API/Page_Visibility_API) has been unprefixed ({{bug("812086")}}).
 - {{domxref("TextDecoder")}} and {{domxref("TextEncoder")}} have been added. Note that the implementation and spec of these evolved and have been changed in Firefox 19 ({{bug("764234")}}).
-- `HTMLMediaElement.src` has been separate in two properties: the standard `src` property, dealing with {{domxref("DOMString")}}, and the prefixed `mozSrcObject` property, dealing with [media streams](/en-US/docs/Web/API/Media_Streams_API) ({{bug("792665")}}).
+- `HTMLMediaElement.src` has been separate in two properties: the standard `src` property, dealing with string, and the prefixed `mozSrcObject` property, dealing with [media streams](/en-US/docs/Web/API/Media_Streams_API) ({{bug("792665")}}).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/18/index.md:35]: Use plural here
```suggestion
- `HTMLMediaElement.src` has been separate in two properties: the standard `src` property, dealing with strings, and the prefixed `mozSrcObject` property, dealing with [media streams](/en-US/docs/Web/API/Media_Streams_API) ({{bug("792665")}}).
```

File: files/en-us/mozilla/firefox/releases/18/index.md [hunk 2]
```diff
@@ -32,9 +32,9 @@ Firefox 18 was released on January 8, 2013. This article lists key changes that
...
 - Support for [transferable objects.](/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#passing_data_by_transferring_.c2.a0ownership_%28transferable_objects%29)
-- The {{domxref("Screen.lockOrientation()")}} method now supports an `Array` of {{domxref("DOMString")}} as argument ({{bug("784549")}}.
+- The {{domxref("Screen.lockOrientation()")}} method now supports an `Array` of string as argument ({{bug("784549")}}.
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/18/index.md:37]:
```suggestion
- The {{domxref("Screen.lockOrientation()")}} method now supports an `Array` of strings as argument ({{bug("784549")}}.
```

File: files/en-us/web/api/bluetoothremotegattcharacteristic/uuid/index.md [hunk 3]
```diff
@@ -16,13 +16,13 @@ browser-compat: api.BluetoothRemoteGATTCharacteristic.uuid
 {{SeeCompatTable}}
 
 The **`BluetoothRemoteGATTCharacteristic.uuid`** read-only
-property returns {{domxref("DOMString")}} containing the UUID of the characteristic, for
+property returns string containing the UUID of the characteristic, for
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothremotegattcharacteristic/uuid/index.md:19]:
```suggestion
property returns a string containing the UUID of the characteristic, for
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**